Tests/SonarQube.MSBuild.Tasks.IntegrationTests/Infrastructure/BuildUtilities.cs

```diff
@@ -28,52 +28,16 @@ internal static class BuildUtilities
         private const string UserImportBeforePropertyName = "ImportUserLocationsByWildcardBeforeMicrosoftCommonTargets";
         private const string UserImportAfterPropertyName = "ImportUserLocationsByWildcardAfterMicrosoftCommonTargets";
 
-        #region Public methods
-
-        /// <summary>
-        /// Creates and returns a valid, initialized MSBuild ProjectRootElement for a new project in the
-        /// specified parent folder
-        /// </summary>
-        /// <param name="projectDirectory">The folder in which the project should be created</param>
-        /// <param name="preImportProperties">Any MSBuild properties that should be set before any targets are imported</param>
-        public static ProjectRootElement CreateValidProjectRoot(TestContext testContext, string projectDirectory, IDictionary<string, string> preImportProperties, bool isVBProject = false)
-        {
-            ProjectDescriptor descriptor = CreateValidProjectDescriptor(projectDirectory);
-            ProjectRootElement projectRoot = CreateInitializedProjectRoot(testContext, descriptor, preImportProperties, isVBProject);
-            return projectRoot;
-        }
-
-        /// <summary>
-        /// Creates and returns a valid, initialized MSBuild ProjectRootElement for a new project in the
-        /// specified parent folder with the specified project file name
-        /// </summary>
-        /// <param name="projectDirectory">The folder in which the project should be created</param>
-        /// <param name="projectFileName">The name of the project file</param>
-        /// <param name="preImportProperties">Any MSBuild properties that should be set before any targets are imported</param>
-        /// <returns></returns>
-        public static ProjectRootElement CreateValidNamedProjectRoot(TestContext testContext, string projectFileName, string projectDirectory, IDictionary<string, string> preImportProperties)
-        {
-            ProjectDescriptor descriptor = CreateValidNamedProjectDescriptor(projectDirectory, projectFileName);
-            ProjectRootElement projectRoot = CreateInitializedProjectRoot(testContext, descriptor, preImportProperties);
-            return projectRoot;
-        }
-
-        /// <summary>
-        /// Creates and returns a valid project descriptor for a project in the supplied folders
-        /// </summary>
-        public static ProjectDescriptor CreateValidProjectDescriptor(string parentDirectory)
-        {
-            return CreateValidNamedProjectDescriptor(parentDirectory, "MyProject.xproj.txt");
-        }
+        #region Project creation helpers
```

Comment:
Comparing the changes in this file is difficult in the GitHub web UI. I've moved some methods around and added regions to group them by purpose.